mdyd-dev/pos-cli
# Update dependencies

## QA
Most important things should work, which i consider:
* deploy
* sync
* gui serve (graphql)
* logs

But given that nothing was updated for a long time, it's a good idea to check every command (and think about tests ;) )
